eox_core/api/data/aggregated_collector/utils.py

```diff
@@ -0,0 +1,100 @@
+"""
+Utility functions for report generation, including query execution
+and integration with the Shipyard API.
```

Comment:
Lets remove this mention of the Shipyard API  here please, I don't think its necessary to mention that service